tools/osx/crosstool/CROSSTOOL.tpl

```diff
@@ -13132,8 +13132,6 @@ toolchain {
         flag: "-Xlinker"
         flag: "-bitcode_verify"
         flag: "-Xlinker"
-        flag: "-bitcode_hide_symbols"
```

Comment:
Why are you removing this?

Reply:
I think this case is really only if you want to hide your symbols from Apple. That has the downside of them not being able to create valid dsyms for you, and their crash reporter not being able to show symbols. I feel like this should be something consumers opt into

Reply:
I was wrong about this, reverting